### ISSUE TITLE
offering option to disable video chat

### ISSUE THREAD

**Issue by Haboubi92** [chamilo/chamilo-lms#2831]
I suggest offering option in configuration setting to disable video chat , the reason for this suggestion is to minimizing resources spent of video chat and reducing the load on the server while in the same time still offering global chat tool which is very useful feature.
![chat](https://user-images.githubusercontent.com/48198054/54025552-e8a74480-41ab-11e9-857b-3ccc5df6969f.PNG)

**Comment by NicoDucou** [comment]
That'sa good idea and it has already been added ;) and will be available in the 1.11.10 release of Chamilo : 
```
// Hide global chat video
//$_configuration['hide_chat_video'] = false;

```

**Comment by ywarnier** [comment]
Actually, video will be disabled by default in 1.11.10 as it introduces a security issue
